%YAML:1.0
---
cloud_header_node:
    params:
        plane:            
            distanceThreshold: 4.               #float

       